Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    firespring_dev_commands (2.2.7)
+    firespring_dev_commands (2.3.0.pre.alpha.1)
```

Comment:
Your target is fine, but when pushing changes its a good idea to check the current versions available. This should have continued in the 2.2 domain.